core/modules/compare/components/CompareButton.ts

```diff
@@ -0,0 +1,8 @@
+export const CompareButton = {
+  name: 'CompareButton',
+  computed: {
+    compareIsActive () : boolean {
```

Comment:
`isEmpty` with opposite value may be more descriptive (also to change in Vuex) what do you think?

Reply:
changed to isEmpty, but IMO we should change `CompareButton` component to  `isCompareEmpty` and add there `isCompareEmpty` computed property. We use this computed property in two places but can be helpfull in multiple places. What fo you thing?

Reply:
You mean completely remove`CompareButton` or add `isCompareEmpty` component? in both cases i think it should be a vuex action, not a component since it's just a function with return value,